app/torii-providers/stripe.js

```diff
@@ -0,0 +1,26 @@
+import stripeConnect from 'torii/providers/stripe-connect';
+import { alias } from '@ember/object/computed';
+import { inject } from '@ember/service';
+import { configurable } from 'torii/configuration';
+
+function currentUrl() {
```

Comment:
Why do you need this ?

Reply:
If i don't include this. It sends redirect to /create/torii/redirect.html and gives error that it cannot parse it. Since on stripe dashboard I have given it as localhost:4200/torii/redirect.html . So either I change it on my stripe dashboard or change it here (I did this to omit the /create part from url)? Please suggest.

Reply:
@niranjan94 I did some more research on this and here is what is found:
1. From docs of torii they said that
> the redirect URL you register with the OAuth provider(s) that you use should be: your app base URL/torii/redirect.html

Other redirect URIs were deprecated by torii and have potential vulnerability as given [here](https://github.com/Vestorly/torii/blob/master/README.md#oauth-redirects). 

2. Also if I use redirect uri as `http://localhost:4200/create/torii/redirect.html it shows this error message 
![screenshot from 2018-06-13 15-08-54](https://user-images.githubusercontent.com/21259802/41343463-54fb4694-6f1c-11e8-8379-c2ddd563fd6d.png)

Please suggest me if any changes are required.

Reply:
That's fine. What i mean is, why do you need to override it here when it is already done at https://github.com/Vestorly/torii/blob/master/addon/providers/oauth2-code.js#L107. You don't even have to define redirectUri here. It is already part of the super class

Reply:
@niranjan94 The definition given there also takes into account windows.location.pathname which will eventually include `create` and `events/<event_id>/edit/basic-details` (editing created event) before `torii/redirect.html` . So I thought overriding existing function so that redirect uri is always `http://localhost:4200/torii/redirect.html` (removed windows.location.pathname from function).

Reply:
Okay. Couldn't you use 

```js
return `${window.location.origin}/torii/redirect.html` 
```